### ISSUE TITLE
eego sports - no trigger signal

### ISSUE THREAD

**Issue by evko6688** [mne-tools/mne-cpp#480]
@johaenns

Triggers are not received from the Trigger adapter (connected to a PC) connected to the eego in MNE scan

The command window from MNE scan regarding the triggers shows the following error:
QTextStream: No device
QTestStream: No device
No projector specified for these data.
No projector specified for these data.
No projector specified for these data.
... 

No projector specified for these data appears if a new stimulus is presented (e.g. checkerboard changes.

**Comment by johaenns** [comment]
I receive the same notifications, nevertheless the triggers are recorded in my case. Could you check, whether you can see the triggers (channel "STIM") in the recording window and whether they show up if you save the recorded data to disk?

**Comment by evko6688** [comment]
You are right, the triggers are there, just needed to scroll, sorry then the problem changes to, when I add the averaging plug-in it shows no average according to the trigger

**Comment by LorenzE** [comment]
What channel is shown in the QuickControl GUI window, see screenshot below?
 
![image](https://user-images.githubusercontent.com/5374202/74380294-4b8b5d80-4db7-11ea-88a2-f12fd82e13ef.png)

**Comment by johaenns** [comment]
![image](https://user-images.githubusercontent.com/18470885/74417046-5c36e480-4e46-11ea-8bea-08942840d342.png)

This is what it looks like for me. The trigger detection seems to work correctly, the averaging is hard to control, since I'm not performing any measurement.

**Comment by LorenzE** [comment]
Wer you able to receive the trigger channel with your setup?

**Comment by evko6688** [comment]
![image](https://user-images.githubusercontent.com/59090366/75049124-f263ae00-54c9-11ea-9b0e-6edb6d73dd1f.png)

Yes the trigger is visible.

**Comment by evko6688** [comment]
How show the averaged signal look like if it works properly (I mean color etc.)?

**Comment by LorenzE** [comment]
So it seems triggers are coming in. Can you check which color you selected in the Selection tab? Maybe change the background color to black in the Other tab as well.

**Comment by chdinh** [comment]
Old MNE Scan / MNE Analyze report against 2020-era UI/plugin code that has since been substantially reworked. There is no current reproducer on staging. Closing as stale; please reopen with up-to-date reproduction steps if it still occurs.

**Comment by chdinh** [comment]
Thanks for the detailed report. The EEGoSports plugin and the trigger/stim handling have changed considerably since 2020, and we couldn't reproduce the missing-trigger behaviour on current `staging`. Closing for now to keep things tidy — if it still occurs with a recent build, please reopen with the eego setup and trigger-adapter details and we'll take another look.
